tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/proton/ProtonTest.java

```diff
@@ -211,6 +213,25 @@ public void testBrokerContainerId() throws Exception {
    }
 
    @Test
+   public void testBrokerConnectionProperties() throws Exception {
+      if (protocol != 0 && protocol != 3) return; // Only run this test for AMQP protocol
```

Comment:
Can this be an assume statement (this way the test will be marked as skipped if this is not true)